### PR TITLE
flowControl

### PR DESCRIPTION
**Pull request by SurgeArrester**

### CODE REVIEW

File: labs/flowControl/README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ while (lessThanTen) {
 
 We should use `while` statements when we do not know how many times a loop will need to run before terminating. 
 
-{% spoiler "Hint" %}
+{% spoiler "Hint"}
```
Comment (pazz) [inline labs/flowControl/README.md:99]: didn't i fix this before already?

Comment (pazz) [reply labs/flowControl/README.md:99]: This is the reason for lab.cs50.io not working (see also #2 )

**Comment by patricktotzke** [comment]
This looks good to me. We're missing a problem set to check the submission.
Perhaps this lab (and problem set and with it the slug) should now be called "leapYear"?

**Comment by patricktotzke** [comment]
I believe we still want to input strings and not ints in the checks..
Also, I could be wrong but wasn't there a separate exercise to compute leap-years using Boolean combinations? If so, perhaps there should be checks for that too.

**Comment by patricktotzke** [comment]
Math service down?
This is how the lab looks on labs.cs50.io (and github) as of today:

2020-10-20-082343_499x589_scrot

**Review by patricktotzke — APPROVED** [review]

File: labs/flowControl/README.md [hunk 2]
```diff
@@ -42,17 +42,21 @@ else {
 }
 ```
 
+{% next %}
+
 ## Leap Years
 
-In our Gregorian calendar, a leap year is a year that is divisible by $`4`$, except if it is also divisible by $`100`$, in which case it's a leap year only if it's divisible by $`400`$. So $`1980`$, $`2000`$, and $`2016`$ are all leap years, while $`1900`$, $`2001`$ and $`2018`$ are not.
+In our Gregorian calendar, a leap year is a year that is divisible by ![equation](https://latex.codecogs.com/gif.latex?4), except if it is also divisible by ![equation](https://latex.codecogs.com/gif.latex?100), in which case it's a leap year only if it's divisible by ![equation](https://latex.codecogs.com/gif.latex?400). So ![equation](https://latex.codecogs.com/gif.latex?1980), ![equation](https://latex.codecogs.com/gif.latex?2000), and ![equation](https://latex.codecogs.com/gif.latex?2016) are all leap years, while ![equation](https://latex.codecogs.com/gif.latex?1900), ![equation](https://latex.codecogs.com/gif.latex?2001) and ![equation](https://latex.codecogs.com/gif.latex?2018) are not.
```
Comment (patricktotzke) [inline labs/flowControl/README.md:49]: 2020-10-20-082916_887x482_scrot

It looks like their certificates are too old for my browser to accept. This isn't really a durable setup I think..

File: labs/flowControl/README.md [hunk 3]
```diff
@@ -69,89 +73,10 @@ false    // output
...
-
 ## Submission
 
-These programs will be tested automatically against a range of test cases, so ensure that the output matches the examples given otherwise these are guaranteed to fail. 
+These programs will be tested automatically against a range of test cases, so ensure that the output matches the examples given otherwise these are guaranteed to fail.
 
-Once you have completed these exercises submit these in the usual way 
+Once you have completed these exercises submit these in the usual way
 
 check50 liv-ac-uk/comp122/2021/labs/variables
```
Comment (patricktotzke) [inline labs/flowControl/README.md:82]: Check your code using

```
check50 liv-ac-uk/comp122/2021/labs/flowControl
```

Submit via

```
submit50 liv-ac-uk/comp122/2021/labs/flowControl
```

**Review by patricktotzke — CHANGES_REQUESTED** [review]

**Comment by pazz** [comment]
(added as "leapYears" with small mods)

**Pull request closed without merging**